src/test/resources/tiny_infra

```diff
@@ -1 +0,0 @@
-../../../examples/tiny_infra/
```

Comment:
The symlink was replaced by the real files. the folder `src/test/resources/tiny_infra` has to be reverted.